MAUI/Popup/working-with-mvvm.md

```diff
@@ -0,0 +1,444 @@
+---
+layout: post
+title: MVVM in .NET MAUI Popup control | Syncfusion
+description: Learn here all about MVVM support in Syncfusion .NET MAUI Popup (SfPopup) control, its elements and more.
+platform: MAUI
+control: SfPopup
+documentation: ug
+---
+
+# Working with MVVM in MAUI Popup (SfPopup)
+
+`SfPopup` can be used in MVVM architecture applications easily. In the below example [SfPopup.IsOpen](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Popup.SfPopup.html#Syncfusion_Maui_Popup_SfPopup_IsOpen) property is bound to a property in the `ViewModel` based on which the popup is opened or closed. Refer the below code example to display popup in MVVM.
+
+In the below code snippet, note that the BindingContext is set for the page and the property (IsOpen) of the ViewModel is bound to the [SfPopup.IsOpen](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Popup.SfPopup.html#Syncfusion_Maui_Popup_SfPopup_IsOpen).

```

Comment:
In the following code sample, note that the BindingContext is set for the page, and the property (IsOpen) of the ViewModel is bound to the [SfPopup.IsOpen](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Popup.SfPopup.html#Syncfusion_Maui_Popup_SfPopup_IsOpen).